mage/git.go

```diff
@@ -0,0 +1,99 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mage
+
+import (
+	"github.com/pkg/errors"
+
+	"sigs.k8s.io/release-utils/command"
+)
+
+const (
+	gitConfigNameKey    = "user.name"
+	gitConfigNameValue  = "releng-ci-user"
+	gitConfigEmailKey   = "user.email"
+	gitConfigEmailValue = "nobody@k8s.io"
+)
+
+func CheckGitConfigExists() (bool, error) {
+	userName := command.New(
+		"git",
+		"config",
+		"--global",
+		"--get",
+		gitConfigNameKey,
+	)
+
+	stream, err := userName.RunSuccessOutput()
+	if err != nil {
+		return false, errors.Wrap(err, "failed to get git user's name")
```

Comment:
When wrapping the errors, we  generally use gerunds as messages get chained together behind the source cause:
```suggestion
		return false, errors.Wrap(err, "getting git user.name")
```
Consider these two:

> _Error failed to check git config: **failed to get git user's name**: git command not found_

vs:
> _Error checking git config: **getting git user's name**: git command not found_